man/rmd/bake_dependent_roles.Rmd

```diff
@@ -0,0 +1,55 @@
+## Effects of non-standard roles
+
+
+Recipes can label and retain column(s) of your data set that should not be treated as outcomes or predictors. For example, a unique identifier column or some other ancillary data that can be used to troubleshoot issues during model development.
+
+As an example, you might have a date column that is transformed into features using [step_date()] or [step_holiday()]. 
+
+```{r, results = "hide", message = FALSE, warning = FALSE}
+  library(tidymodels)
+  tidymodels_prefer()
+
+  data(Chicago, package = "modeldata")
+  chi_rec <- recipe(ridership ~ ., data = Chicago) %>% 
+    # Since `date` is symbolically on the right-hand side of the formula, it
+    # is treated as a predictor (i.e., has a role of "predictor")
+    step_holiday(date) %>% 
+    step_date(date)
```

Comment:
Can we use something here that is not `date`? I think this gets really muddy because it is sort of a predictor and you are using it to create predictors, and we have better ways to not keep the original variable around. I think `tate_text` would be a better option for these docs.

Reply:
I've spent a bit more time on this, and I do think we can do better than the `date` example for being clear about how to use these other roles.

Reply:
This is exactly the application that has tried people up (us included). It seems most relevant. I can add another example though.